packages/ui/src/stories/DashboardShowcase.stories.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useContext, useEffect } from 'react'
+import { BrowserRouter } from 'react-router-dom'
+import { ThemeContext } from 'styled-components'
+
+import { Button } from '../Button'
+import { Icon } from '../Icon'
+import {
+  ColumnLayout,
+  Page,
+  PageLayout,
+  StackLayout,
+  usePageLayout,
+} from '../Layouts'
+import { CellProps, Table } from '../Table'
+import { Text } from '../Text'
+
+export default {
+  title: 'Showcases/Dashboard',
+  component: Page,
+}
+
+const defaultColumns = {
+  assets: {
+    header: 'Assets',
+    Cell: ({ data: { assets } }: CellProps) => <div>{assets}</div>,
+    alignText: 'left',
+  },
+  totalValueDeposited: {
+    header: 'Total value deposited',
+    Cell: ({ data: { totalValueDeposited } }: CellProps) => (
+      <span>{totalValueDeposited}</span>
+    ),
+  },
+  totalBorrowed: {
+    header: 'Total borrowed',
+    Cell: ({ data: { totalBorrowed } }: CellProps) => (
+      <span>{totalBorrowed}</span>
+    ),
+  },
+  depositAPY: {
+    header: 'Deposit APY',
+    Cell: ({ data: { depositAPY } }: CellProps) => <span>{depositAPY}</span>,
+  },
+  borrowAPY: {
```

Comment:
on design i see that this last column is centerAlign, not right
![image](https://user-images.githubusercontent.com/22115870/179393742-6347f54d-204c-458a-b22b-f42d4c5107aa.png)